src/utils/parseViyaDebugResponse.ts

```diff
@@ -16,10 +16,15 @@ export const parseSasViyaDebugResponse = async (
   requestClient: RequestClient,
   serverUrl: string
 ) => {
+  // On viya 3.5, iframe is like <iframe style="width: 99%; height: 500px" src="..."></iframe>
+  // On viya 4, iframe is like <iframe style="width: 99%; height: 500px; background-color:Canvas;" src=...></iframe>
+
   const iframeStart = response.split(
-    '<iframe style="width: 99%; height: 500px" src="'
+    /<iframe style="width: 99%; height: 500px" src="|<iframe style="width: 99%; height: 500px; background-color:Canvas;" src=/
```

Comment:
I am bit afraid anytime that styles could be changed, so I suggest we use something more generic as a regex.
For example: `/<iframe style=.*src=.*/`

It would match in all cases.